CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+cmake_minimum_required(VERSION 3.0)
+
+# Project is not versioned, so use date of last upstream commit.
```

Comment:
Usually I do `0.0.0` in this case because if they will release for example `1.0.0` then we will have situation when `2018.10.29` is greater than `1.0.0` (see https://semver.org/).

Also `VERSION` is not a requirement neither for CMake install config nor Hunter, so you are free to remove related parts.

Reply:
Got it, thanks. I'll do 0.0.0 then